### PR TITLE
Fixes #25845 - Module stream info for erratum

### PR DESCRIPTION
**Pull request by parthaa**
This commit displays module streams information for erratum.

### CODE REVIEW

**Comment by parthaa** [comment]
```
$ hammer erratum info --id=6
Title:            Duck_Kangaroo_Erratum
Description:      Duck_Kangaro_Erratum description
ID:               6
Errata ID:        RHEA-2012:0059
Reboot Suggested: false
Updated:          2018-07-20
Issued:           2018-01-27
Solution:         
Packages:         duck-0.7-1.noarch
Module Streams:   
 1) Name:     kangaroo
    Stream:   0
    Packages: kangaroo-0.3-1.noarch
```

**Comment by parthaa** [comment]
```
$ hammer host erratum info --id=6 --host-id=2
Title:            Duck_Kangaroo_Erratum
Description:      Duck_Kangaro_Erratum description
ID:               6
Errata ID:        RHEA-2012:0059
Reboot Suggested: false
Updated:          2018-07-20
Issued:           2018-01-27
Solution:         
Packages:         duck-0.7-1.noarch
Module Streams:   
 1) Name:     kangaroo
    Stream:   0
    Packages: kangaroo-0.3-1.noarch
```

**Comment by parthaa** [comment]
Same behavior in the host errata info call
```
$ hammer  host errata info --host-id=1 --id=RHEA-2012:0059
Title:            Duck_Kangaroo_Erratum
Description:      Duck_Kangaro_Erratum description
ID:               6
Errata ID:        RHEA-2012:0059
Reboot Suggested: false
Updated:          2018-07-20
Issued:           2018-01-27
Solution:         
Packages:         duck-0.7-1.noarch
Module Streams:   
 1) Name:     kangaroo
    Stream:   0
    Packages: kangaroo-0.3-1.noarch

```

**Comment by jturel** [comment]
Taking a look :)

File: lib/hammer_cli_katello/erratum_info_command.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      field :id, _("ID")
+      field :errata_id, _("Errata ID")
+      field :reboot_suggested, _("Reboot Suggested")
+      field :updated, _("Updated")
+      field :issued, _("Issued")
+      field :release, _("Release")
+      field :solution, _("Solution")
+      field :packages, _("Packages"), Fields::List
+
+      collection :module_streams, _("Module Streams") do
```
Comment (akofink) [inline lib/hammer_cli_katello/erratum_info_command.rb]:
```suggestion
      collection :module_streams, _("Module Streams"), hide_blank: true do
```

**Pull request merged**